packages/block-directory/src/components/downloadable-block-author-info/index.js

```diff
@@ -12,13 +12,18 @@ function DownloadableBlockAuthorInfo( {
 	return (
 		<Fragment>
 			<span className="block-directory-downloadable-block-author-info__content-author">
-				{ sprintf( __( 'Authored by %s' ), author ) }
+				{ sprintf(
+					/* translators: %s: author name. */
+					__( 'Authored by %s' ),
+					author
+				) }
 			</span>
 			<span className="block-directory-downloadable-block-author-info__content">
 				{ sprintf(
+					/* translators: 1: number of blocks. 2: average rating. */
 					_n(
-						'This author has %d block, with an average rating of %d.',
-						'This author has %d blocks, with an average rating of %d.',
+						'This author has %1$d block, with an average rating of %2$d.',
+						'This author has %1$d blocks, with an average rating of %2$d.',
```

Comment:
Not really something I expect action on as part of this pull request, but I find these specific changes interesting, and perhaps evident of another rule we want to explore:

https://github.com/Automattic/wp-calypso/blob/master/packages/eslint-plugin-wpcalypso/docs/rules/i18n-named-placeholders.md

i.e. Our current configuration is such that these changes were not driven by ESLint rules, but there is a standard here we want to encourage (multiple placeholders being named or numbered?), which we could enforce.

Reply:
I actually have a PR for that, see #20574.

I only changed this particular string as part of this PR because there was no translator comment, but in order to write one, I need to fix the string.